### ISSUE TITLE
Power operation is unsupported

### ISSUE THREAD

**Issue by fubuloubu** [ethereum/viper#363]
### What's your issue about?

Exponents are not currently supported

### How can it be fixed?

Add them.

Test via:
```python
val: public(num)
def __init__():
    self.val = 10**2
```
```python
c = compile(...)
assert c.get_val() == 100
```

#### Cute Animal Picture

![baby corgi](https://i.pinimg.com/736x/8c/10/fb/8c10fb39489477c701dfe9c419ce3766--corgi-puppies-baby-corgi.jpg)

**Comment by DavidKnott** [comment]
I'm working on this now.

**Comment by fubuloubu** [comment]
Closed due to #364
